app/src/main/java/com/example/chrono/main/timer/CircuitCreate.kt

```diff
@@ -182,4 +184,36 @@ class CircuitCreate : BaseActivity() {
             }
         }
     }
+
+    // Floor to a multiple of timeChangeVal
+    private fun floorVal(valToFloor: Int): Int {
+        return (valToFloor / timeChangeVal) * (timeChangeVal)
+    }
+
+    //When decrementing set time, subtract to the closest multiple of timeChangeVal
+    private fun roundTimeDown(valToRound: Int): Int {
+        return if (valToRound % timeChangeVal == 0) {
+            valToRound - timeChangeVal
+        } else {
+            floorVal(valToRound)
+        }
+    }
+
+    //Change SetNum textView text
+    private fun setSetNum(newVal: Int) {
+        bind!!.setNum.setText(newVal.toString())
```

Comment:
One liners don't need to be in a function, defeats the purpose cuz it takes one line to call the function, and it's one line of code that's being called in the function. Also the function takes 3 lines of code whereas the code for it is just one line. Pls change